manifests/plugin/puppetdb/params.pp

```diff
@@ -31,6 +31,9 @@
       fail("${::hostname}: puppetdb_foreman does not support osfamily ${::osfamily}")
     }
   }
-  $address = 'https://localhost:8081/v2/commands'
-  $dashboard_address = 'http://localhost:8080/dashboard'
+  $address = 'https://localhost:8081/pdb/cmd/v1'
+  $dashboard_address = 'http://localhost:8080/pdb/dashboard'
+  $ssl_ca_file       = $::foreman::client_ssl_ca
```

Comment:
`$::foreman` can't be used as defaults here, at least with foreman-installer. You could include foreman::params and then use `$::foreman::params::client_ssl_ca` though.